openlr/openlr_stat/openlr_stat.cpp

```diff
@@ -192,9 +193,25 @@ void WriteAssessmentFile(std::string const fileName, pugi::xml_document const &
   }
 
   pugi::xml_document result;
+  auto segments = result.append_child("Segments");
+  auto const dict = doc.select_node(".//Dictionary").node();
+
+  // Copy namespaces from <Dictionary> to <Segments>
+  for (auto const attr : dict.attributes())
+  {
+    if (strncmp("xmlns", attr.name(), 5) != 0)
+      continue;
+
+    // Don't copy default namespace.
+    if (strlen(attr.name()) == 5)
```

Comment:
All default namespaces and only default namespaces have names of size 5?

Reply:
I guess so, cause default namespace is defined like so: xmlns="...". Any other namespace is defined this way: xmlns:namespace="...".